app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
Yeah, I had mentioned wanting to move to using FormRequests for validation stuff before... now might be a good time as @DFurnes suggests ;)